src/services/layerHelper.js

```diff
@@ -81,7 +81,8 @@ ngeo.LayerHelper.prototype.createBasicWMSLayer = function(sourceURL,
   const source = new ol.source.ImageWMS({
     url: sourceURL,
     params,
-    serverType: olServerType
+    serverType: olServerType,
+    crossOrigin: 'anonymous'
```

Comment:
I think that we should add an option for that, and in a second time id should be configurable by the OGC server :-)